### PR TITLE
Logo in the nav bar. 

### PR DESCRIPTION
**Pull request by sheenachhabradev**
Issue - logo in the nav bar #76 
https://github.com/Real-Dev-Squad/website-welcome/issues/76

1. Removed navbar.html. 
2. Added home logo element to all html files. 
3. Added css for home logo.

### CODE REVIEW

File: code-of-conduct.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 	</HEAD>
 	<BODY>
 		<nav>
+					<div class="homeLogo">
+						<a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
+					</div>
```
Comment (swarajpure) [inline code-of-conduct.html:16]: Not properly indented. Can we please remove those extra spaces?

File: css/pages/navbar.css [hunk 2]
```diff
@@ -90,7 +94,7 @@ nav li:hover a{
```
Comment (swarajpure) [inline css/pages/navbar.css:94]: Can we please get rid of these extra spaces (I know, they're not a part of your PR, but still)

**Review by swarajpure — APPROVED** [review]
Very minor indentation changes requested. Thank you! 🎉

File: code-of-conduct.html (hunk 1 above)
Comment (sheenachhabradev) [reply code-of-conduct.html]: Extra spaces removed.

File: css/pages/navbar.css (hunk 2 above)
Comment (sheenachhabradev) [reply css/pages/navbar.css]: @swarajpure  updated.

File: code-of-conduct.html [hunk 3]
```diff
@@ -11,20 +11,20 @@
...
-           <ul class="menu">
-            <li><a href="index.html">Home</a></li>
-            <li class="active"><a href="code-of-conduct.html">Code of Conduct</a></li>
-            <li><a href="faq.html">FAQ</a></li>
-            <li><a href="discord.html">Discord</a></li>
-           </ul>
-        </nav>
+			<div class="homeLogo">
+				<a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
+			</div>
```
Comment (swarajpure) [inline code-of-conduct.html:16]: I was talking about the indentation on line number 14, 15 and 16 😄

**Review by swarajpure — CHANGES_REQUESTED** [review]

File: code-of-conduct.html (hunk 3 above)
Comment (sheenachhabradev) [reply code-of-conduct.html]: Replaced tab with 2 spaces.

File: css/pages/navbar.css [hunk 4]
```diff
@@ -2,11 +2,10 @@
   margin:0px;
   padding:0px;
  }
- 
-  
+   
 nav {
     display: grid;
-    grid-template-columns: 1fr 2fr 1fr;
+    grid-template-columns: 0.3fr 2fr 1fr;
```
Comment (ankushdharkar) [inline css/pages/navbar.css:8]: Isn't grid an overkill for a linear navbar?

File: css/pages/navbar.css [hunk 5]
```diff
@@ -139,7 +134,4 @@ nav li:hover a{
 
  li.active a{
   color:white;
-} 
-  
-       
-
+}
```
Comment (ankushdharkar) [inline css/pages/navbar.css:137]: Nit: Endline missing

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: code-of-conduct.html [hunk 6]
```diff
@@ -11,17 +11,20 @@
...
-            <li><a href="discord.html">Discord</a></li>
-           </ul>
-        </nav>
+      <div class="homeLogo">
+        <a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
+      </div>
+      <input type="checkbox" id="nav-toggle">
+      <label for="nav-toggle" class="burger-menu">
+      <img src="img/logos/burger.jpg" width="35px" height="auto" alt="">
+      </label>
```
[the remark this answers is not in the document]
Comment (sheenachhabradev) [reply code-of-conduct.html:20]: @ankushdharkar  
Should it be 2 spaces / a tab ? 
Have noticed a difference in structure when viewing in VS Studio or in Git.

**Comment by Rucha1499** [comment]
This issue is solved! Closing this PR. Ref : https://welcome.realdevsquad.com/

**Pull request closed without merging**